app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -160,7 +160,6 @@ class HomeActivityTest {
   @get:Rule
   val accessibilityTestRule = AccessibilityTestRule()
 
-  @get:Rule
```

Comment:
@vrajdesai78 this is what's causing your CI failures. I suggest making sure that you're running changed tests locally & often to make sure you know what your changes work as expected.